### PR TITLE
Valid json in api docs

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -10469,7 +10469,7 @@ Querying general information
 	      "backend_default_arguments": {
 	          "max-logfiles-num": 3,
```
Comment (yabirgb) [inline docs/api.rst:10470]: the `data_directory` key is missing a `,` at the end

File: docs/api.rst [hunk 2]
```diff
@@ -10469,7 +10469,7 @@ Querying general information
...
 	          "max-size-in-mb-all-logs": 300,
-	          "sqlite-instructions": 5000,
+	          "sqlite-instructions": 5000
 	      },
```
Comment (yabirgb) [inline docs/api.rst:10473]: This `,` is also extra

File: docs/api.rst (hunk 1 above)
Comment (yabirgb) [reply docs/api.rst:10470]: `accept_docker_risk` is missing a `"`

Comment (yabirgb) [reply docs/api.rst:10470]: `accept_docker_risk` is missing a `"`

Comment (yabirgb) [reply docs/api.rst:10470]:
```
{
  "result": {
    "version": {
      "our_version": "1.0.3",
      "latest_version": "1.0.4",
      "download_url": "https://github.com/rotki/rotki/releases/tag/v1.0.4"
    },
    "data_directory": "/home/username/.local/share/rotki/data",
    "log_level": "DEBUG",
    "accept_docker_risk": false,
    "backend_default_arguments": {
      "max-logfiles-num": 3,
      "max-size-in-mb-all-logs": 300,
      "sqlite-instructions": 5000
    }
  },
  "message": ""
}
```

This one has all the issues fixed

File: docs/api.rst (hunk 2 above)
Comment (LefterisJP) [reply docs/api.rst:10473]: what?

Comment (LefterisJP) [reply docs/api.rst]: I ran it through a linter. It had also more stuff. It should be okay now.

**Review by yabirgb — APPROVED** [review]
I give you my approval

**Pull request merged**